jenkins/scripts/integration_test_env.sh

```diff
@@ -64,4 +64,10 @@ CAPM3BRANCH="${CAPM3BRANCH:-${CAPM3RELEASEBRANCH}}"
 
 # Container image registry value to override the default value in m3-dev-env
 export CONTAINER_REGISTRY=${CONTAINER_REGISTRY:-"registry.nordix.org/quay-io-proxy"}
+
+#NOTE (elfosardo): this can be removed once https://github.com/metal3-io/metal3-dev-env/pull/1473 merges
```

Comment:
We need to keep Docker Hub for some images like Kind, so this comment is incorrect.